### PR TITLE
Add Azure Blob videos

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds videos to these articles:
https://help.sumologic.com/docs/send-data/collect-from-other-data-sources/azure-blob-storage/
https://help.sumologic.com/docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage/

Issue number: None. Found out about the new videos from "The Weekly Practice" email on Aug 30 from Jay Schwegler.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage.md [hunk 1]
```diff
@@ -17,6 +17,21 @@ This section has instructions for configuring a pipeline for shipping logs avail
...
+        height="480px"
+        id="myId"
+        className="video-container"
+        display="initial"
+        position="relative"
+        allow="accelerometer; autoplay=1; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+        />
+
+import Iframe from 'react-iframe';
```
Comment (JV0812) [inline docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage.md:33]: Can we move this import Iframe command to top of the page and keep it with the import usebaseUrl.

File: docs/send-data/collect-from-other-data-sources/azure-blob-storage/index.md [hunk 2]
```diff
@@ -14,6 +14,21 @@ This solution is for newly created blobs only (not for existing blobs). 
...
+        height="480px"
+        id="myId"
+        className="video-container"
+        display="initial"
+        position="relative"
+        allow="accelerometer; autoplay=1; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+        />
+
+import Iframe from 'react-iframe';
```
Comment (JV0812) [inline docs/send-data/collect-from-other-data-sources/azure-blob-storage/index.md:30]: Can we move this import Iframe command to top of the page.

**Review by JV0812 — APPROVED** [review]
Minor fix

File: docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage.md (hunk 1 above)
Comment (jpipkin1) [reply docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage.md]: Good suggestion, @JV0812! Done.

File: docs/send-data/collect-from-other-data-sources/azure-blob-storage/index.md (hunk 2 above)
Comment (jpipkin1) [reply docs/send-data/collect-from-other-data-sources/azure-blob-storage/index.md]: Done!

**Pull request merged**